code/game/objects/items/crayons.dm

```diff
@@ -771,6 +775,7 @@
 	name = "cyborg spraycan"
 	desc = "A metallic container containing shiny synthesised paint."
 	charges = -1
+	stun_delay = 50 // 3 seconds longer than the knockdown time
```

Comment:
i know its not exactly important but isn't this comment inaccurate now

Reply:
woops thanks